client/src/pages/Login.js

```diff
@@ -62,8 +62,8 @@ const Login = () => {
 
     return (
         <Grid container className={classes.vh100}>
```

Comment:
as the screen resizes, the container changes to a vertical view from a horizontal view, can you try using flex direction or display inline-block to tackle it